test/unit/display_utils_spec.js

```diff
@@ -324,4 +325,64 @@ describe("display_utils", function () {
       );
     });
   });
+
+  describe("renderRichText", function () {
+    const makeBag = s => new Set(s.split(/[<>/ ]+/).filter(x => x));
```

Comment:
I'm a bit confused about the terminology here, as in, what is a "bag" in this context? Trying it out in the console makes me wonder: would `splitParts` perhaps be a more descriptive name here?

Is there a reason why we can't just compare the full HTML string as-is without any splitting since that would be even easier?